bowlingScore.test.js

```diff
@@ -0,0 +1,42 @@
+const BowlingGame = require('./bowlingScore')
+
+describe('bowlingGame', () => {
+  it('makes a bowling class', () => {
+    const bowlingGame = new BowlingGame();
+  })
+  it('han an empty array for frames', () => {
+    const bowlingGame = new BowlingGame();
+    expect(bowlingGame.frames).toEqual([])
+  })
+  // needs a double as don't want to actualy add a number to the array, just test it is being added
+  // it('roll once', () => {
+  //   const bowlingGame = new BowlingGame();
+  //   expect(bowlingGame.knockedPins(1)).toEqual(1)
```

Comment:
If you commented out this, for when pushing to the frames array. You could change the expect to something like 
`expect(bowlingGame.frames).toEqual([1])` Last bit checking that it is an array in the end.